scripts/ASTImportTest.sh

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env bash
 
+set -e
+
 # Bash script to test the ast-import option of the compiler by
 # first exporting a .sol file to JSON, then loading it into the compiler
 # and exporting it again. The second JSON should be identical to the first
-
-REPO_ROOT=$(readlink -f "$(dirname "$0")"/..)
+READLINK=readlink
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    READLINK=greadlink
```

Comment:
What packages is this contained in?

Reply:
@axic `greadlink` is part of `coreutils`, installable via `brew install coreutils`.

Reply:
Can that be added to the dependencies list in the documentation? Or well, into the some developer docs which explains how to run all the tests.